pkg/catalog/catalog.go

```diff
@@ -23,9 +23,9 @@ import (
 	"github.com/hashicorp/go-multierror"
 	lru "github.com/hnlq715/golang-lru"
 	"github.com/rs/xid"
+	"github.com/treeverse/lakefs/modules/block/factory"
```

Comment:
```suggestion
	blockfactory "github.com/treeverse/lakefs/modules/block/factory"
```

Reply:
Let's say this happens, and we have both imports (which we currently don't) - 
Why the `modules` one should be `blockfactory` and the other just `factory`?

First, I don't think such convention is currently necessary.
And if it is necessary - please have a convention that reflects the fact that it's a module. I'd say `moduleblockfactory`, but it's too much.

Bottom line, I think we should leave it for now, since it doesn't make things clearer IMO.

Reply:
What do you mean lets say it happens - we're going to have a module called `config/factory` when you import both, they will both be `factory`. One of the will be automatically renamed but I believe the convention should be to explicitly name them!

Reply:
Ahhh, I thought you meant a module and a local with the same name.
Agree.
Updated.